test/features/proof_of_address.feature

```diff
@@ -0,0 +1,150 @@
+@browser
+Feature: Proof of address
+
+  Scenario Outline: Test file upload for PoC docs
+    Given I navigate to the SDK with PoA feature enabled
+    When I navigate to poa document upload screen after selecting <poa_document>
+# currently we don't run validation of the uploaded documents hence we upload id
+    When I upload national_identity_card
+    Then I should see poa_upload_confirmation_title (POA)
+    Then I should see poa_upload_confirmation_subtitle (POA)
+
+    Examples:
+      | poa_document                  |
+      | bank_building_statement_cell  |
+      | utility_bill_cell             |
+      | council_tax_letter_cell       |
+      | benefits_letter_cell          |
+
+  Scenario: Test presence of the UI elements of the second screen of the PoA flow
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    Then I should see verify_uk_address_title (POA)
+    Then I should see need_a_doc_text (POA)
+    Then I should see poa_requirements (POA)
+
+  Scenario: Test navigation to the document selection screen
+    Given I navigate to the SDK with PoA feature enabled
+    When I click on primary_button (SDK)
+    When I click on start_verification_button (POA)
+    Then I should see document_selection_screen_title (POA)
+    Then I should see document_selection_screen_subtitle (POA)
+
+  Scenario: Test presence of the UI elements of the bank/building society statement cell
```

Comment:
From line 33 to 62 there is a lot of repetition that could be avoided

Reply:
Done :)